windows/RNCamera/RCTCameraModule.cs

```diff
@@ -0,0 +1,451 @@
+            if (mediaCapture == null)
+            {
+                promise.Reject("No camera found.");
+                return;
+            }
+
+            var isSupported = mediaCapture.VideoDeviceController.FlashControl.Supported;
+            promise.Resolve(isSupported);
+        }
+
+        public void OnSuspend()
+        {
+            var recordingTask = _recordingTask;
+            if (recordingTask != null)
+            {
+                _recordingCancellation.Cancel();
+                recordingTask.Wait();
+            }
+        }
+
+        public void OnResume()
+        {
+        }
+
+        public void OnDestroy()
+        {
+        }
+
+        private async Task CapturePhotoAsync(MediaCapture mediaCapture, JObject options, IPromise promise)
+        {
+            var encoding = ImageEncodingProperties.CreateJpeg();
+            var target = options.Value<int>("target");
+            if (target == CameraCaptureTargetMemory)
+            {
+                using (var stream = new InMemoryRandomAccessStream())
+                {
+                    await mediaCapture.CapturePhotoToStreamAsync(encoding, stream).AsTask().ConfigureAwait(false);
+                    var data = await GetBase64DataAsync(stream).ConfigureAwait(false);
+                    promise.Resolve(new JObject
+                    {
+                        { "data", data },
+                    });
+                }
+            }
+            else
+            {
+                var storageFile = await GetOutputStorageFileAsync(MediaTypeImage, target).AsTask().ConfigureAwait(false);
+                await mediaCapture.CapturePhotoToStorageFileAsync(encoding, storageFile).AsTask().ConfigureAwait(false);
+                await UpdateImagePropertiesAsync(storageFile, options);
+                promise.Resolve(new JObject
+                {
+                    { "path", storageFile.Path },
+                });
+            }
+        }
+
+        private async Task RecordAsync(CameraForView cameraForView, JObject options, IPromise promise, CancellationToken token)
+        {
+            var mediaCapture = cameraForView.MediaCapture;
+            var taskCompletionSource = new TaskCompletionSource<bool>();
+            using (var cancellationTokenSource = new CancellationTokenSource())
+            using (token.Register(cancellationTokenSource.Cancel))
+            using (cancellationTokenSource.Token.Register(async () => await StopRecordingAsync(mediaCapture, taskCompletionSource)))
+            {
+                var quality = (VideoEncodingQuality)options.Value<int>("quality");
+                var encodingProfile = MediaEncodingProfile.CreateMp4(quality);
+
+                mediaCapture.AudioDeviceController.Muted = options.Value<bool>("audio");
+
+                // TODO: test orientation
+                var orientation = options.ContainsKey("orientation") ? options.Value<int>("orientation") : cameraForView.Orientation;
```

Comment:
Delete.

Reply:
![image](https://user-images.githubusercontent.com/25733576/39284843-89864542-493f-11e8-815b-013741e7bce1.png)
this properties is null when i'm try to call the camera!! Can u help me??